novem/api_ref.py

```diff
@@ -72,17 +73,24 @@ def __init__(self, **kwargs: Any) -> None:
         # api root should always be supplied in the result
         self._api_root = config["api_root"]
 
+        env_token = os.getenv("NOVEM_TOKEN")
+
         if config.get("token", None):
             assert config["token"]
             self.token = config["token"]
             self._session.auth = ("", self.token)
 
+        elif env_token is not None:
+            self.token = env_token
+            self._session.auth = ("", self.token)
+
```

Comment:
I think a token from env should take precedence over the default config approach, since it's often explicitly added to the command

Reply:
After further thought explicit > implicit